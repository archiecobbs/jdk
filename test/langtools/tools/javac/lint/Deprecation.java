/*
 * @test /nodynamiccopyright/
 * @bug 4821359
 * @summary Add -Xlint flag
 * @author gafter
 *
 * @compile/fail/ref=Deprecation.out -XDrawDiagnostics  -Xlint:deprecation -Werror -XDshould-stop.ifError=WARN Deprecation.java
 */

/** @deprecated */
class A {
}

class B extends A {
}
