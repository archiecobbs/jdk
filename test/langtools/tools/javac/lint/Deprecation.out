Deprecation.java:11:1: compiler.warn.missing.deprecated.annotation
Deprecation.java:14:17: compiler.warn.has.been.deprecated: A, compiler.misc.unnamed.package
- compiler.err.warnings.and.werror
1 error
2 warnings
